README.md

```diff
@@ -45,19 +45,20 @@ node {
 ### withAvd
```

Comment:
Can you update `withAvd.txt` as well, that is used to generate some documentation within Jenkins.

Reply:
Done.